File: backend/src/main/java/com/lawencon/ticketms/dto/ticket/TicketInsertReqDto.java
package com.lawencon.ticketms.dto.ticket;

import java.util.List;

public class TicketInsertReqDto {
	private String title;
	private String content;
	private String ticketPriorityCode;
	private Long productCustomerId;
	private List<FileInsertReqDto> file;
	public String getTitle() {
		return title;
	}
	public void setTitle(String title) {
		this.title = title;
	}
	public String getContent() {
		return content;
	}
	public void setContent(String content) {
		this.content = content;
	}
	public String getTicketPriorityCode() {
		return ticketPriorityCode;
	}
	public void setTicketPriorityCode(String ticketPriorityCode) {
		this.ticketPriorityCode = ticketPriorityCode;
	}
	public Long getProductCustomerId() {
		return productCustomerId;
	}
	public void setProductCustomerId(Long productCustomerId) {
		this.productCustomerId = productCustomerId;
	}
	public List<FileInsertReqDto> getFile() {
		return file;
	}
	public void setFile(List<FileInsertReqDto> file) {
		this.file = file;
	}
}

File: frontend/src/app/dto/ticket/ticket.insert.req.dto.ts
import { FileInsertReqDto } from "./file.insert.req.dto"

export class TicketInsertReqDto {

    private title! : string
	private content! : string
	private ticketPriorityCode! : string
	private productCustomerId! : number
	private file! : FileInsertReqDto[]

    public get _title(): string {
        return this.title;
    }

    public set _title(_title: string) {
        this.title = _title;
    }

    public get _content(): string {
        return this.content;
    }

    public set _content(_content: string) {
        this.content = _content;
    }

    public get _ticketPriorityCode(): string {
        return this.ticketPriorityCode;
    }

    public set _ticketPriorityCode(_ticketPriorityCode: string) {
        this.ticketPriorityCode = _ticketPriorityCode;
    }

    public get _productCustomerId(): number {
        return this.productCustomerId;
    }

    public set _productCustomerId(_productCustomerId: number) {
        this.productCustomerId = _productCustomerId;
    }

    public get _file(): FileInsertReqDto[] {
        return this.file;
    }

    public set _file(_file: FileInsertReqDto[]) {
        this.file = _file;
    }

}